include/LayerPlan.h

```diff
@@ -778,6 +778,12 @@ class LayerPlan : public NoCopy
      */
     void applyBackPressureCompensation();
 
+    /*!
+     * If enabled, applies the gradual flow acceleration splitting, that improves printing quality when printing at very high speed
+     * with a bowden extruder.
```

Comment:
Nitpicking, but direct drive also benefits from this, although to a lesser extend.

Reply:
Ok, didn't know that, I'll refine the comment